common/views/components/SearchFilters/SearchFilters.tsx

```diff
@@ -13,6 +13,7 @@ type Props = {
   changeHandler: () => void;
   filters: Filter[];
   linkResolver: (params: ParsedUrlQuery) => LinkProps;
+  newStyle?: boolean;
```

Comment:
Could we rename to `isNewStyle` to enforce that it's boolean?

Reply:
done